### PR TITLE
I149 clean the repo

### LINKED ISSUE #149: Clean the repo
Clean the repo:

- [x] remove .env file
- [x] remove all but one icon libs
- [x] pin dependencies including dev
- [x] remove brand colours in tailwind config
- [x] refactor colours in tailwind config

### PR DESCRIPTION
**Pull request by NicholasChoong**
## Change Summary
- [x] remove .env file
- [x] remove all but one icon libs
- [x] pin dependencies including dev
- [x] remove brand colours in tailwind config
- [x] refactor colours in tailwind config

heroicons v1 -> v2
dark-red -> primary-dark
cream -> gray-50
dark-gray -> gray-500
zinc-50 -> white
zinc-200 -> zinc-100
zinc-300 -> zinc-200

# Related Issue

- Resolve #149

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -38,8 +38,7 @@ sitemap.xml
 sitemap-*.xml
 
 # environmonet
-# .env
-.env.local
```
Comment (scratchclaggy) [inline .gitignore:42]: Shouldn't we keep `.env.local` in the ignore?

File: package.json [hunk 2]
```diff
@@ -29,49 +29,42 @@
...
+    "@heroicons/react": "2.0.13",
+    "@next/bundle-analyzer": "13.0.5",
+    "class-variance-authority": "0.3.0",
+    "firebase": "9.14.0",
+    "next": "12.3.4",
+    "next-pwa": "5.6.0",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
+    "react-hook-form": "7.39.5",
+    "react-icons": "4.6.0",
```
Comment (scratchclaggy) [inline package.json:41]: Just confirming, we're keeping react-icons + heroicons?

File: src/components/UI/logoSvg/index.tsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  'M24 12.073c0-6.627-5.373-12-12-12s-12 5.373-12 12c0 5.99 4.388 10.954 10.125 11.854v-8.385H7.078v-3.47h3.047V9.43c0-3.007 1.792-4.669 4.533-4.669 1.312 0 2.686.235 2.686.235v2.953H15.83c-1.491 0-1.956.925-1.956 1.874v2.25h3.328l-.532 3.47h-2.796v8.385C19.612 23.027 24 18.062 24 12.073z'
+const Google =
+  'M12.48 10.92v3.28h7.84c-.24 1.84-.853 3.187-1.787 4.133-1.147 1.147-2.933 2.4-6.053 2.4-4.827 0-8.6-3.893-8.6-8.72s3.773-8.72 8.6-8.72c2.6 0 4.507 1.027 5.907 2.347l2.307-2.307C18.747 1.44 16.133 0 12.48 0 5.867 0 .307 5.387.307 12s5.56 12 12.173 12c3.573 0 6.267-1.173 8.373-3.36 2.16-2.16 2.84-5.213 2.84-7.667 0-.76-.053-1.467-.173-2.053H12.48z'
+const Microsoft =
+  'M0 0v11.408h11.408V0zm12.594 0v11.408H24V0zM0 12.594V24h11.408V12.594zm12.594 0V24H24V12.594z'
+const SolidEmojiSad =
+  'M10 18a8 8 0 100-16 8 8 0 000 16zM7 9a1 1 0 100-2 1 1 0 000 2zm7-1a1 1 0 11-2 0 1 1 0 012 0zm-7.536 5.879a1 1 0 001.415 0 3 3 0 014.242 0 1 1 0 001.415-1.415 5 5 0 00-7.072 0 1 1 0 000 1.415z'
+
+const Logo = (props: svgProps) => {
+  const svgPath = useCallback(() => {
```
Comment (scratchclaggy) [inline src/components/UI/logoSvg/index.tsx:22]: The `props.name` should never change, so we don't need `useCallback`?

File: tailwind.config.js [hunk 4]
```diff
@@ -10,27 +10,16 @@ module.exports = {
...
-      googleyellow: '#FBBC05',
-      googlered: '#EA4335',
-      applegrey: '#A2AAAD',
-      microsoftblue: '#00A4EF',
-      yahoopurple: '#430297',
-      'dark-gray': '#7B7B7B',
-      'zinc-200': '#F2F2F2',
-      'zinc-300': '#D9D9D9',
-      'zinc-50': '#FFFFFF'
+      zinc: colors.zinc
```
Comment (scratchclaggy) [inline tailwind.config.js:22]: Do we need both 'gray' and 'zinc'? https://tailwindcss.com/docs/customizing-colors#default-color-palette

**Review by scratchclaggy — COMMENTED** [review]
All good, just left a few comments for you. Also I noticed that `logoSvg` uses a different naming convention / directory structure from the rest of the components, is that style the way going forward?

File: .gitignore (hunk 1 above)
Comment (NicholasChoong) [reply .gitignore:42]: I removed that `.env.local` because it has already been mentioned above on line 27

**Review by scratchclaggy — APPROVED** [review]
Looks good dude :rocket:

**Comment by scratchclaggy** [comment]
@NicholasChoong Have you done a `grep` of the codebase for all of the removed tailwind colors?

**Comment by NicholasChoong** [comment]
> @NicholasChoong Have you done a `grep` of the codebase for all of the removed tailwind colors?

yea i did

**Review by scratchclaggy — APPROVED** [review]
Should be good now :rocket:

**Review by skyheat — APPROVED** [review]

**Pull request merged**